lib/spack/spack/test/database.py

```diff
@@ -719,13 +719,13 @@ def test_external_entries_in_db(mutable_database):
     assert not rec.spec.external_modules
 
     rec = mutable_database.get_record("externaltool")
-    assert rec.spec.external_path == os.sep + os.path.join("path", "to", "external_tool")
+    assert rec.spec.external_path == os.path.sep + os.path.join("path", "to", "external_tool")
```

Comment:
This change is fine but AFAIK `os.sep` == `os.path.sep` so this is just for clarity right?

Reply:
Yes.

Reply:
Honestly, I'm kinda shocked that `os.path.sep` and `os.pathsep` were allowed to be two different conceptual concepts by the python folks.